modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -1826,6 +1851,74 @@ m| dbms.cluster.uncordonServer(server :: STRING)
 m|WRITE
 |===
 
+[[procedure_dbms_cluster_deallocateDatabase]]
+[role=label--enterprise-edition label--admin-only]
+.dbms.cluster.deallocateDatabase()
```

Comment:
This procedure seems internal in the codebase `internal.dbms.cluster.deallocateDatabase`. Is this correct?